addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/FormatStringScanRule.java

```diff
@@ -276,13 +272,11 @@ public Map<String, String> getAlertTags() {
 
     @Override
     public int getCweId() {
-        // The CWE id
         return 134;
```

Comment:
Add URL here too? (or remove from other places?)

Reply:
You're right, on the earlier one I probably should have put the description/title instead of the URL. I think that's what we have most places.